recipes/click-spinner/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "click-spinner" %}
+{% set version = "0.1.8" %}
+{% set sha256 = "67b5af5e825faf82a4fc6cda77c58359abe716fb1c9bc12cc7bea9a0cae1fc8e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - add-missing-tests.patch
+    - add-missing-license.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - click
+    - pytest
+    - six
+  imports:
+    - click_spinner
+  source_files:
+    - tests  # this will copy the external tests folder to the temp test dir
+  commands:
+    - py.test tests
+
+about:
+  home: https://github.com/click-contrib/click-spinner 
+  license: MIT
+  license_file: LICENSE
```

Comment:
If the license is not being packaged, please open a PR in the ``click-spinner`` project, but for now you can just put the license along with this recipe instead of patching it

Reply:
Please, do the same for tests

Reply:
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-section

https://conda-forge.org/docs/meta.html?highlight=license#packaging-the-license-manually

Reply:
@marcelotrevisani Thanks for the links! Knew about the license file but somehow missed that tests can be included directly with the recipe as well, nice 👍 I will add your suggestions asap, thank you for reviewing :)